agent/bench-scripts/postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Should this script live in the `postprocess` directory.  Why not make it a function inside `pbench-user-benchmark`?